src/components/Poster/Poster.js

```diff
@@ -0,0 +1,12 @@
+import './Poster.css';
+import React from 'react';
+
+const Poster = ({data}) => {
+    return (
+        <article className='poster'>
```

Comment:
I think our class names should be capitalizing the first letter, I just know that App's is and I vaguely remember this from class